src/NowController.ts

```diff
@@ -0,0 +1,25 @@
+import {SlackBot, SlackMessage} from 'botkit';
+import axios from 'axios';
+
+export class NowController {
+  static async run(bot: SlackBot, msg: SlackMessage) {
+    const response = await axios.request({
+      url: process.env.KURUO_URL,
+      method: 'get',
+      responseType: 'stream'
+    });
+
+    const payload = {
+      channels: msg.channel,
+      filename: 'kuruo.jpg',
+      title: 'kuruo_now',
+      file: response.data,
+    };
+
+    bot.api.files.upload(payload, (err, _) => {
```

Comment:
If `KURUO_URL` is a static URL (ex: `https://kuruo.com/img/kuruo.jpg`), uploading is unnecessary.

```js
bot.reply(msg, `${process.env.KURUO_URL}`);
```

Reply:
I understood that `KURUO_URL` should not reveal to users.